src/plugins/orangehrmCorePlugin/Service/ConfigService.php

```diff
@@ -51,6 +51,7 @@ class ConfigService
     public const KEY_INSTANCE_IDENTIFIER_CHECKSUM = 'instance.identifier_checksum';
     public const KEY_SENDMAIL_PATH = 'email_config.sendmail_path';
     public const KEY_LDAP_SETTINGS = 'ldap_settings';
```

Comment:
```suggestion
    public const KEY_LDAP_SETTINGS = 'admin.ldap_settings';
```
:thinking: what do you think? Seems this is my mistake

Reply:
yeah saw this one and wondered why there is no prefix. Shall I add the prefix